server/heartbeat/index.ts

```diff
@@ -57,12 +62,30 @@ const timerTrigger: AzureFunction = async function (
 
   await DB.setActiveUsers(activeUsers)
 
+  // This is a hack because sometimes people, for unknown reasons, get removed from activeUsers but *not* room presences
+  const roomIds = Object.values(roomData).map((r) => r.id)

```

Comment:
I think it's fine for now, since this won't meaningfully affect performance, but for your own knowledge, `Object.keys()` would give us what we want here (since the keys are the room IDs)